### DISCUSSION TITLE
Naming folders based on different layouts

### DISCUSSION THREAD

**Discussion by amir-rh99** [vercel/next.js#57395] [Help]
Do you believe it's a sound decision to select folder names according to the distinct page layouts? For instance, in situations where certain pages don't require a footer and necessitate a distinct layout, is it advisable to create a separate layout designated specifically for pages featuring a footer?

![image](https://github.com/vercel/next.js/assets/64532154/78cdaf4d-dfd5-41f4-b2b0-0620af3d5207)

**Comment by landsman** [comment]
I would rather hide footer in the layout based on URL or other "flag" if it's the only difference in UI.
